8/changelog.yaml

```diff
@@ -71,8 +71,6 @@ releases:
     release_date: '2023-05-23'
   8.0.0:
     changes:
-      deprecated_features:
-        - The netapp.aws collection is considered unmaintained and will be removed
```

Comment:
The `deprecated_features` entry is missing. Please re-add it and re-build everything to make sure it shows up in the RST changelog and porting guide that are included in the tarball (you can also manually check that; the RST changelog and porting guide in this PR contain this entry even though it's not in here).